app/components/JobCard.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { PropTypes } from 'react';
```

Comment:
Let's just get into the habit of using `createClass` for components. You can probably do this yourself, based on the examples I gave :).

If components get complex at all (have any methods, such as `delete`), then this kind of component can't really be used anymore. It's technically more performant, but not by enough for the headaches of needing to refactor later.